be/src/storage/lake/delta_writer.cpp

```diff
@@ -323,6 +325,19 @@ Status DeltaWriterImpl::open() {
     return Status::OK();
 }
 
+Status DeltaWriterImpl::check_op_cols() {
+    // If more than one column has __op, then return error
+    if (_tablet_schema->keys_type() == KeysType::PRIMARY_KEYS && _slots != nullptr) {
+        int op_count = std::count_if(_slots->begin(), _slots->end(), [](const SlotDescriptor* slot) {
+            return slot != nullptr && slot->col_name() == "__op";
+        });
+        if (op_count > 1) {
+            return Status::InvalidArgument("PK table can't use __op as column name");
```

Comment:
is it possible to print the table name or id here?

Reply:
After talk with Yanbo, I remove check on BE side.